### PR TITLE
Airbyte integration

### PR DESCRIPTION
**Pull request by Santhin**
`<description>`

Resolves `<issue nr here>`

---
Keep in mind:
- [ ] Documentation updates
- [ ] [Changelog](CHANGELOG.md) updates

### CODE REVIEW

File: data_pipelines_cli/cli_commands/deploy.py [hunk 1]
```diff
@@ -43,9 +43,11 @@ def __init__(
         dags_path: Optional[str],
         provider_kwargs_dict: Optional[Dict[str, Any]],
         datahub_ingest: bool,
+        airbyte_ingest: bool,
```
Comment (p-pekala) [inline data_pipelines_cli/cli_commands/deploy.py:46]: maybe something more generic like: enable_ingestion: bool

File: data_pipelines_cli/cli_commands/deploy.py [hunk 2]
```diff
@@ -156,12 +166,19 @@ def _sync_bucket(self) -> None:
     default=False,
     help="Whether to ingest DataHub metadata",
 )
+@click.option(
+    "--airbyte-ingest",
```
Comment (p-pekala) [inline data_pipelines_cli/cli_commands/deploy.py:170]: --enable-ingestion ?

File: tests/test_airbyte_utils.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+from unittest.mock import patch
+import unittest
+from data_pipelines_cli.airbyte_utils import env_replacer
+import os
+
+class AirbyteUtilsTest(unittest.TestCase):
```
Comment (p-pekala) [inline tests/test_airbyte_utils.py:6]: problably test for factory is missine?

File: data_pipelines_cli/cli_commands/deploy.py [hunk 4]
```diff
@@ -156,12 +167,19 @@ def _sync_bucket(self) -> None:
     default=False,
     help="Whether to ingest DataHub metadata",
 )
+@click.option(
+    "--enable-ingest",
```
Comment (paszynski) [inline data_pipelines_cli/cli_commands/deploy.py:171]: From user perspective there might be some confusion. Should I set `--enable-ingest` flag when I would like to set `--datahub-ingest`?

Comment (p-pekala) [reply data_pipelines_cli/cli_commands/deploy.py:171]: You know what, I'm thinking now... we don't want to control it with flag. Rather we would like to turn on/off based on ingestion.yml. Thanks to that we won't need to change CICD template.

**Review by p-pekala — CHANGES_REQUESTED** [review]

File: data_pipelines_cli/cli_commands/deploy.py (hunk 4 above)
Comment (paszynski) [reply data_pipelines_cli/cli_commands/deploy.py:171]: Hm interesting simplification idea. You mean the deploy command would behave as follows:
1. Lookup `enabled` value in ingestion.yml . 
2. If set to true, lookup file indicated by `engine` variable and perform factory actions

I assume the same algorithm should apply to BI files as well?

Comment (p-pekala) [reply data_pipelines_cli/cli_commands/deploy.py:171]: exactly, I already wrote comment for Pawel chagnes

**Review by p-pekala — APPROVED** [review]

**Pull request merged**